### PR TITLE
[DEV-3764] Adds NS Branded Logo and Updates Landing Screen

### PR DESCRIPTION
**Pull request by cdanwards**
## DESCRIPTION

We currently don't have the NS Branded Logo in the app and the Apollos Logo is shown on load.

### What does this PR do, or why is it needed?

Screen Shot 2019-05-13 at 11 31 07 AM

This PR adds the NS branding icons in the app and updates accordingly.
The icons are : `LeafOutline`, `LeafSolid`, `LeafLined`, and `LeafLogo`

### How do I test this PR?

Make sure you're signed out and launch onboarding. Make sure that the first landing screen has our logo.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-3764](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3764)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by redreceipt — DISMISSED** [review]
it's on the initial splash screen but needs to be on every other onboarding screen and the connect screen when you're logged out.

Screen Shot 2019-05-13 at 12 12 01 PM
Screen Shot 2019-05-13 at 12 11 23 PM
Screen Shot 2019-05-13 at 12 10 23 PM
Screen Shot 2019-05-13 at 12 07 53 PM

**Comment by cdanwards** [comment]
@redreceipt I got this one wrapped up now with all the right icons.

File: packages/newspringchurchapp/src/index.js [hunk 1]
```diff
@@ -22,12 +22,22 @@ import Prayer from './prayer';
...
 import UserWebBrowser from './user-web-browser';
 import Onboarding from './ui/Onboarding';
+import NSIcon from './ui/NSIcon';
 
 const AppStatusBar = withTheme(({ theme }) => ({
   barStyle: 'dark-content',
   backgroundColor: theme.colors.paper,
 }))(StatusBar);
 
+const BrandIcon = withTheme(({ theme }) => ({
```
Comment (richarddubay) [inline packages/newspringchurchapp/src/index.js:32]: I'm confused why these changes are needed? There's no match to this in core and their icons seem to work fine. In my testing this is obviously controlling stuff because the icons on the onboarding slides seem to only change size based on the size set here and not the size that is already set in the slide file. I'm not sure why we would need to override that.

**Review by richarddubay — COMMENTED** [review]
From a style perspective I think the icons are too big. Maybe that's okay ... someone else can give their opinion. I think I'd like to see all the onboarding icons move down to a `baseUnit *5` instead of a 6. And if we're following core then the icon on the Connect logged out view is smaller than the ones used in onboarding. In that case that particular icon could probably be dropped down to a `baseUnit *4`.

File: packages/newspringchurchapp/src/index.js (hunk 1 above)
Comment (cdanwards) [reply packages/newspringchurchapp/src/index.js:32]: @richarddubay for this one specifically, it _has_ to be defined here to be passed in for the SMS Login screens. In Core, this is set in the `ui-auth` package which we cannot edit.

**Comment by cdanwards** [comment]
@richarddubay that's fair on the icon size. Honestly, I just went with what I liked.. sooo personal preference.

**Review by IsaacHardy — APPROVED** [review]
I agree with @richarddubay here on Icon sizes. HOWEVER, when we finally meet with creative on app UI, we will need to lean into their expertise on how NewSpring brands Icon sizes. We want to have a close resemblance to core, but certain "branding" guidelines will have to be synergized with NewSpring.

PR looks good to merge.
